move CRUD workshop to workshops repo (#168)
CodeYourFuture/CYF-Workshops: crud/readme.md

1. `npm run dev`
1. Open Postman
1. Make a GET request to `http://localhost:3000/albums`

### 2. `GET /albums/:albumId should return a single album (that matches the passed albumId)`

Sometimes, we do not want to list all the information in one request, maybe we only want to get the information related to a single album. Imagine if we have a page to display the details of one album. We could call the server and get all albums then filter the one we need client-side. It would be more effective to tell the server to just return the one album we are interested in.

We will now add a new endpoint to return only a single album GET /albums/:albumId. In this case, albumId will tell us what album we can return. The call will be `GET /albums/10` and that will return the album with that has `albumId: "10"`.

This endpoint has something different. The endpoint `/albums/:albumId` has a dynamic part. The `albumId` will vary depending on what the client sends.

In `server.js`, create a `GET /albums/:albumId` endpoint that returns a single album. The albumId will be passed as a parameter in the URL.

```js
app.get("/albums/:albumId", (req, res) => {
  const albumId = req.params.albumId;

>   const album = albumsData.find((album) => album.albumId === albumId);

Reviewer: ```suggestion
 // now find the given album from the `albumsData` using the `albumId`
```

>   res.send(album);

Reviewer: ```suggestion
 // finally send the album you found back to the client
```

});
```

#### 🧪 Run and test

1. Save your changes

> 1. Make a GET request to `http://localhost:3000/albums/10`

Reviewer: ```suggestion
1. Make a GET request to `http://localhost:3000/albums/10`
1. Try changing the id in the URL and calling the endpoint again. What do you see?
```
